Mathlib/GroupTheory/QuotientGroup/Defs.lean

```diff
@@ -216,6 +225,23 @@ theorem map_comp_map {I : Type*} [Group I] (M : Subgroup H) (O : Subgroup I) [M.
     (map M O g hg).comp (map N M f hf) = map N O (g.comp f) hgf :=
   MonoidHom.ext (map_map N M O f g hf hg hgf)
 
+@[to_additive (attr := simp)]
+theorem comap_map_mk' (N H : Subgroup G) [N.Normal] :
+    Subgroup.comap (mk' N) (Subgroup.map (mk' N) H) = N ⊔ H :=
+  by simp [Subgroup.comap_map_eq, sup_comm]
```

Comment:
Any particular reason to choose this order for `H` and `N`? It seems to save a `simp` argument if we do it the other way, without breaking downstream usage.
```suggestion
    Subgroup.comap (mk' N) (Subgroup.map (mk' N) H) = H ⊔ N :=
  by simp [Subgroup.comap_map_eq]
```

Reply:
Suggestion adopted, thanks!